neurom/features/neurite_features.py

```diff
@@ -109,6 +110,19 @@ def number_of_sections_per_neurite(neurites, neurite_type=TreeType.all):
     return (_sec.count(n) for n in neurites if _ttc(neurite_type)(n))
 
 
+@as_neurite_list
+def total_length_per_neurite(neurites, neurite_type=TreeType.all):
+    '''Get an iterable with the total length of a neurite for a given neurite type'''
+    return (sum(_sec.length(ss) for ss in isection(n))
```

Comment:
OK great! But please don't spend too much time on it. If it proves to be to complicated, just let me know.